sigstore-java/src/main/java/dev/sigstore/tuf/Updater.java

```diff
@@ -234,16 +235,23 @@ void verifyDelegate(
       Map<String, Key> publicKeys,
```

Comment:
github really doesn't want me commenting on non changed lines or something, but line 230 @throws is I guess  out of date?